lib/spack/spack/cmd/env.py

```diff
@@ -568,7 +568,7 @@ def env_revert(args):
     # Check that both the spack.yaml and the backup exist, the inform user
     # on what is going to happen and ask for confirmation
     if not os.path.exists(manifest_file):
-        msg = "cannot fine the manifest file of the environment [file={0}]"
+        msg = "cannot find the manifest file of the environment [file={0}]"
```

Comment:
I guess Spack couldn't fine a file too :laughing: (thanks for catching the typo)